ports/geckolib/wrapper.rs

```diff
@@ -49,7 +49,8 @@ use style::selector_impl::ElementExt;
 use style::sink::Push;
 use url::Url;
 
-pub type NonOpaqueStyleData = *mut RefCell<PrivateStyleData>;
+pub type NonOpaqueStyleData = RefCell<PrivateStyleData>;
+pub type NonOpaqueStyleDataPtr = *mut NonOpaqueStyleData;
```

Comment:
It was just a leak, and we didn't notice. We'd still free the memory just fine, but wouldn't realize that there was a destructor to run.